force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -347,22 +404,29 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<String> getHighPaidEmployees() {
         // Initialize a Map
-        Map<String, Integer> employeeSalaries = new Map<String, Integer>();
-
+        Map<String, Integer> highSalary = new Map<String, Integer>();
```

Comment:
There are also ways to populate a map statically in a single declaration; look into this. Nothing wrong per se with the approach below, but it can be good to know the syntax to declare and initialize a map with certain values.